zowex/src/main.rs

```diff
@@ -570,6 +564,7 @@ fn start_daemon(njs_zowe_path: &str) -> String {
     let daemon_arg = "--daemon";
     match Command::new(njs_zowe_path.to_owned())
         .arg(daemon_arg)
+        .envs(get_zowe_env())
         .stdout(Stdio::null())
```

Comment:
Just curious...
What happens if we do `.stdin(Stdio::null())` ?

Reply:
According to Rust doc for `Stdio::null`:
> This stream will be ignored. This is the equivalent of attaching the stream to `/dev/null`.

I believe we want to do this for stdout/stderr so they don't get written to the console twice, but for stdin it shouldn't be necessary.